pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -209,9 +212,24 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			}
 		}
 
-		if err := r.reconcileCrashRetention(namespace, cephCluster, cephVersion); err != nil {
-			return reconcile.Result{}, err
+		k8sVersion, err := r.context.Clientset.Discovery().ServerVersion()
+		if err != nil {
+			return reconcile.Result{}, errors.Wrap(err, "failed to fetch get version")
+		}
+
+		minMajorVersionCronV1 := "1"
+		minMinorVersionCronV1 := "21"
+
+		if k8sVersion.Major == minMajorVersionCronV1 && k8sVersion.Minor == minMinorVersionCronV1 {
```

Comment:
There are utility functions like `Atleat`, `LessThan` etc that can be used here.